src/components/ResultFiltersAndShare/ResultFiltersAndShare.tsx

```diff
@@ -15,14 +17,28 @@ function ResultFiltersAndShare({
 }: ResultFiltersAndShareProps) {
   const { state } = useContext(QuizContext);
   const matchedOptions = state?.quiz?.matchedOptions || [];
+  const {
+    text = '',
+    is_active: isActive,
+    items_separator: itemsSeparator,
+    last_separator: lastSeparator,
+  } = state?.quiz?.resultsConfig?.desktop?.response_summary || {};
```

Comment:
getting a type error here `Property 'response_summary' does not exist on type 'ViewportResultsConfig' `

Reply:
you should execute `npm i `, as there are recent changes from internal libraries related to types